docs/en/reference/working-with-objects.rst

```diff
@@ -250,6 +250,11 @@ as follows:
 -  If X is a detached entity, an exception will be thrown on
    flush.
 
+.. caution::
+
+    Do not pass detached entities to the persist operation. The persist operation always
+    considers entities that are not yet known to this EntityManager as NEW.
```

Comment:
I don't think NEW was capitalized for emphasis, but because it is a reference to this:

https://github.com/doctrine/orm/blob/8c259ea5cb632dbb57001b2262048ae7fa52b102/lib/Doctrine/ORM/UnitOfWork.php#L87

Let's make that clearer by using code markup:
```suggestion
    considers entities that are not yet known to this EntityManager as ``NEW``.
```

Reply:
Yepp, that makes sense.

Reply:
Yes I agree.

Am wondering if there are policies about linking the code in the documentation? Or is it to be avoided?

Would it makes sense to link ``NEW`` to the [2.7:UnitOfWork.php#L76](https://github.com/doctrine/orm/blob/2.7/lib/Doctrine/ORM/UnitOfWork.php#L76), to give more context?
My only concern is about eventual commits moving the code to another line and breaking the link.

In that last case the link is to be avoided and maybe we could add details regarding that ``NEW`` word, thrown out without much context.

> Do not pass detached entities to the persist operation. The persist operation always considers entities that are not yet known to the EntityManager as ``NEW`` (refer to the available states  inside ``UnitOfWork``).

or keep consistency with the rest of the documentation and be more discorsive 

> Do not pass detached entities to the persist operation. The persist operation always considers entities that are not yet known to the ``EntityManager`` as new entities (refer to the ``STATE_NEW``  inside the ``UnitOfWork``).


Also note changing to *the* rather than *this EntityManager*, since we are not inside the class.

Reply:
A quick grep shows no links to the code, probably for the reason you stating, although using permalink might mitigate that. I like the second version you propose better.

Reply:
I though about permalinks but that would point a specific commit, while the (impossible) ideal would be to point the branch.

I'll go for the second version then; I like it more too, and it gives enough hints on how to find the correct line.